lib/puppet/util/windows/eventlog.rb

```diff
@@ -0,0 +1,114 @@
+require 'puppet/util/windows'
```

Comment:
I think I'm going to move toward trying to make this file stand alone / completely independent of puppet, and then try requiring it directly from daemon.rb. Not ideal - I dislike having external things requiring puppet's internals (bit us recently with the whole vendor/semantic thing) but since this daemon.rb does live in puppet itself and is exercised indirectly by our acceptance tests I expect any regression would be caught that way.